wordpress-mobile/gutenberg-mobile
# Pass the Gallery v2 Flag over from the editor

### Related PRs
`WordPress-Android`: https://github.com/wordpress-mobile/WordPress-Android/pull/15129
`WordPress-FluxC-Android`: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2065
`gutenberg`: https://github.com/WordPress/gutenberg/pull/33544

### Description
This PR adds the Gallery v2 Flag to the EditorTheme to be passed down to the Editor

To test:
Check https://github.com/WordPress/gutenberg/pull/33544

PR submission checklist:
